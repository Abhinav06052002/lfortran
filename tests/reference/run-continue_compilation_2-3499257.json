{
    "basename": "run-continue_compilation_2-3499257",
    "cmd": "lfortran --continue-compilation --no-color {infile}",
    "infile": "tests/errors/continue_compilation_2.f90",
    "infile_hash": "33d5fef257122bcd9a97860f31b13cfd24870d1e6fb504af9de477e0",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-continue_compilation_2-3499257.stdout",
    "stdout_hash": "09cbe68047489867302494fc1677052657f17c23407106fbec512eeb",
    "stderr": "run-continue_compilation_2-3499257.stderr",
    "stderr_hash": "79a8cf7341dcc2be6bc39554c8e450a9541f3168dde2a0b6f7a29dc0",
    "returncode": 1
}