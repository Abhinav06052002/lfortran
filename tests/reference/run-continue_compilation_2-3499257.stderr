semantic error: shape argument specified in c_f_pointer even though fptr is not an array.
  --> tests/errors/continue_compilation_2.f90:39:38
   |
39 |     call c_f_pointer(queries_1, y_1, [2])
   |                                      ^^^ 

semantic error: shape array passed to c_f_pointer must be of rank 1 but given rank is 2
  --> tests/errors/continue_compilation_2.f90:44:38
   |
44 |     call c_f_pointer(queries_2, y_2, shape)
   |                                      ^^^^^ 

semantic error: Cannot assign to a constant variable
  --> tests/errors/continue_compilation_2.f90:49:5
   |
49 |     x = 1
   |     ^^^^^ assignment here
   |
32 |     integer, parameter :: x = 2
   |                           ~~~~~ declared as constant

semantic error: More actual than formal arguments in procedure call
  --> tests/errors/continue_compilation_2.f90:56:33
   |
56 |     call myCircle%calculateArea(circleArea, 12)
   |                                 ^^^^^^^^^^^^^^ 

semantic error: Invalid argument `end` supplied
  --> tests/errors/continue_compilation_2.f90:62:5
   |
62 |     CLOSE(end=200)
   |     ^^^^^^^^^^^^^^ 

semantic error: The first argument of `cmplx` intrinsic must be present
  --> tests/errors/continue_compilation_2.f90:67:9
   |
67 |     a = cmplx(y = 2)
   |         ^^^^^^^^^^^^ 
